boards/p8z77-m_pro/p8z77-m_pro.config

```diff
@@ -0,0 +1,36 @@
+# Configuration for Asus P8Z77-M Pro
+CONFIG_LINUX_CONFIG=config/linux-p8z77-m_pro.config
+CONFIG_COREBOOT_CONFIG=config/coreboot-p8z77-m_pro.config
+
+export CONFIG_COREBOOT=y
+export CONFIG_COREBOOT_VERSION=4.17
+export CONFIG_LINUX_VERSION=4.14.62
+
+CONFIG_CRYPTSETUP2=y
+CONFIG_FLASHROM=y
+CONFIG_FLASHTOOLS=y
+CONFIG_GPG2=y
+CONFIG_KEXEC=y
+CONFIG_UTIL_LINUX=y
+CONFIG_LVM2=y
+CONFIG_MBEDTLS=y
+CONFIG_PCIUTILS=y
+CONFIG_POPT=y
+CONFIG_QRENCODE=y
+CONFIG_TPMTOTP=y
+CONFIG_CAIRO=y
+CONFIG_FBWHIPTAIL=y
+CONFIG_HOTPKEY=y
```

Comment:
Board should be renamed with -hotp to make clear a Nitrokey pro/storage or librem key is required for remote attestation.

Reply:
good point I guess i can make two versions